templates/zerver/help/mention-a-user-or-group.md

```diff
@@ -72,3 +72,4 @@ streams](/help/stream-notifications).
 
 * [Restrict wildcard mentions](/help/restrict-wildcard-mentions)
 * [Quote and reply](/help/quote-and-reply)
+* [View your mentions](/help/view-your-mentions)
```

Comment:
We should probably cross-link "View your mentions" in `/help/pm-mention-alert-notifications` too.